pkg/util/abiutil/abiutil.go

```diff
@@ -20,3 +21,34 @@ func MustParseABI(json string) abi.ABI {
 	}
 	return val
 }
+
+func ConvertType(in interface{}, proto interface{}) (out interface{}, err error) {
```

Comment:
Some documentation and tests would be great to have. The same for the function below.

Reply:
I dont think these functions are being used anywhere?

Reply:
@swarmHaseeb ^^^